scripts/setup_gitpod.sh

```diff
@@ -5,7 +5,7 @@
 # This is in {} so the pyenv and pre-commit steps can run in the background while docker starts
 # A downside is that progress messages from docker and pre-commit may be mixed together.
 {
-    pyenv install
+    # pyenv install
```

Comment:
Looks like we still need this for gitpod so that it uses python 3.9.4 like the rest of our environment. It _might_ be good enough to use whatever version is there, but would rather keep it consistent.